models/classes/search/tasks/UpdateClassInIndex.php

```diff
@@ -50,19 +51,17 @@ public function __construct(IndexIteratorFactory $indexIteratorFactory = null)
     }
 
     /**
-     * @param $params
-     *
-     * @return Report
-     *
      * @throws common_exception_MissingParameter
+     * @throws common_exception_Error
      */
     public function __invoke($params): Report
     {
         if (empty($params) || empty($params[0])) {
             throw new common_exception_MissingParameter();
         }
 
-        $searchService = $this->getServiceLocator()->get(Search::SERVICE_ID);
+        /** @var SearchProxy $searchService */
+        $searchService = $this->getServiceLocator()->get(SearchProxy::class);
```

Comment:
We need to stick to SERVICE_ID

```suggestion
        $searchService = $this->getServiceLocator()->get(SearchProxy::SERVICE_ID);
```